client/templates/module-template/tests/test-template.ts

```diff
@@ -0,0 +1,10 @@
+import * as uvu from "uvu"
+import * as assert from "uvu/assert"
+
+const SuiteName = uvu.suite("SuiteName")
```

Comment:
```suggestion
import { suite } from "uvu"
import assert from "uvu/assert"

const SuiteName = uvu.suite("SuiteName")
```
Same suggestion here as above